manifests/a/Azul/Zulu/12/JDK/12.3.11/Azul.Zulu.12.JDK.locale.en-US.yaml

```
@@ -0,0 +1,34 @@
+# Created with YamlCreate.ps1 v2.4.1 Dumplings Mod $debug=QUSU.CRLF.7-4-2.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Azul.Zulu.12.JDK
+PackageVersion: 12.3.11

```

```suggestion
PackageVersion: "12.3"
```
Please update file path to match this change.